client/my-sites/plugins/plugin-details.jsx

```diff
@@ -353,12 +357,27 @@ function PluginDetails( props ) {
 										status="is-warning"
 										showDismiss={ false }
 									>
-										<NoticeAction href="https://wordpress.com/support/incompatible-plugins/">
+										<NoticeAction
+											href={ localizeUrl( 'https://wordpress.com/support/incompatible-plugins/' ) }
+										>
 											{ translate( 'More info' ) }
 										</NoticeAction>
 									</Notice>
 								) }
 
+								{ ! isMaintained && (
+									<Notice showDismiss={ false } status="is-warning">
+										{ translate(
+											'This plugin seems outdated and not maintained anymore. We advise that you {{a}}search{{/a}} for another similar plugin.',
+											{
+												components: {
+													a: <a href={ `/plugins/${ selectedSite?.slug ?? '' }` } />,
+												},
+											}
+										) }
+									</Notice>
+								) }
+
```

Comment:
Have you tried putting this message in the top right after https://github.com/Automattic/wp-calypso/blob/1b90390f1aa21968364f6993eaa508ae12ca0c92/client/my-sites/plugins/plugin-details.jsx#L322-L339 ?

As an aside, this seems to be the 4th way in which we show notices in the plugin details 🤪

Reply:
It's a nitpick though. Consolidating all plugins notices could be a good 20% issue

Reply:
I agree, I am going to create a task to refactor that extracting that logic to another component

Reply:
Task created [Refactor: Consolidate all plugin notices to an external component](https://github.com/Automattic/wp-calypso/issues/81314)](https://github.com/Automattic/wp-calypso/issues/81314)